### PR TITLE
Fix for resetting background color on hover and focus state of a disabled flavored button.

### PR DESCRIPTION
**Pull request by denisahac**
Fix for #9021

### CODE REVIEW

**Comment by Silvester20** [comment]
Works great for a normal download, thanks for that! But unfortunately this PR doesn't work for the custom build download. Or actually I probably should say this PR does work but somehow it is not included in the "foundation.min.css" from the [custom build download](http://foundation.zurb.com/sites/download.html/).

Can you take a look at it @kball ?

**Comment by kball** [comment]
@Silvester20 I just tried doing a download of the full from there and it appeared to work fine... can you give me more details about the options you used for the download?

**Comment by Silvester20** [comment]
@kball Yes, I didn't made any change to the custom download. Just go to the [download page](http://foundation.zurb.com/sites/download.html/) and scroll to bottom and press the big orange button "Download Custom Build". Then I took the minified version of the CSS file.

**Comment by kball** [comment]
Thanks @Silvester20, that helped me find it.  There was a caching problem in the customizer and in some cases it was still serving out 6.2.3.  has been fixed.

File: scss/components/_button.scss [hunk 1]
```diff
@@ -136,12 +136,12 @@ $button-opacity-disabled: 0.25 !default;
...
 /// Adds disabled styles to a button by fading the element, reseting the cursor, and disabling pointer events.
-@mixin button-disabled {
+@mixin button-disabled($color: $primary-color) {
   opacity: $button-opacity-disabled;
   cursor: not-allowed;
 
   &:hover, &:focus {
-    background-color: $button-background;
+    background-color: $color;
     color: $button-color;
```
Comment (stianpr) [inline scss/components/_button.scss]: Shouldn't this use `foreground($color, $button-color-alt, $button-color)`? if the button background color is very light? The secondary color is usually gray so we don't want the font color to be white then...

**Pull request merged**